### DISCUSSION TITLE
Add custom link to menu

### DISCUSSION THREAD

**Discussion by AlessandroMinoccheri** [directus/directus#5519] [Archive]
Hi all,
is it possible to add a custom link inside the menu?

if so how?

If I have to create a custom layout, can I see an example out from the documentation? 
Because I think that examples from the documentation are not sufficient for me to understand how to achieve my goal.

I only need to add a link inside the menu to an external resource for my customer.

Thanks

**Comment by benhaynes** [comment]
Which "menu"?

https://docs.directus.io/guides/roles/#customizing-the-module-navigation
https://docs.directus.io/guides/roles/#customizing-the-collection-navigation

**Reply by AlessandroMinoccheri** [reply]
I think that https://docs.directus.io/guides/roles/#customizing-the-module-navigation is good enough.

I can add links for a specific role and could be fine :)

Thanks.
